data/spells/scripts/attack/avalanche.lua

```diff
@@ -7,9 +7,9 @@ local area = createCombatArea(AREA_CIRCLE3X3)
 setCombatArea(combat, area)
 
 function onGetFormulaValues(cid, level, maglevel)
-	min = -((level / 5) + (maglevel * 1.2) + 7)
-	max = -((level / 5) + (maglevel * 2.85) + 16)
-	return min, max
+	min = (level / 5) + ((maglevel * 1.2) + 7)
+	max = (level / 5) + ((maglevel * 2.85) + 16)
+	return -min, -max
 end
```

Comment:
Oh, yeah, I'm aware of that, I just thought of a clearer code instead of a shorter one (especially to emphasize the level quotient (or power "(level/5)") of the spell damage.